modules/cats-effect2/src/main/scala/pureconfig/module/catseffect/package.scala

```diff
@@ -0,0 +1,115 @@
+package pureconfig.module
+
+import java.io.OutputStream
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Path}
+
+import scala.language.higherKinds
+import scala.reflect.ClassTag
+
+import cats.data.EitherT
+import cats.effect.{Blocker, ContextShift, Resource, Sync}
+import cats.implicits._
+import com.typesafe.config.ConfigRenderOptions
+
+import pureconfig._
+import pureconfig.error.ConfigReaderException
+
+package object catseffect {
```

Comment:
In order to be extra clear that this is a backport, I'd rather use a separate `catseffect2` package for this. Having artifacts with different (org, name) pairs containing the same package is confusing and may lead to runtime problems since one does not necessarily evict the other. 

(using both `catseffect` and `catseffect2` modules in the same project is most likely an error anyway, but at least it will be clear if that happens accidentally)